stdlib_list/lists/3.9.txt

```diff
@@ -42,8 +92,17 @@ copyreg
 crypt
 csv
 ctypes
+ctypes._aix
+ctypes._endian
+ctypes.macholib
+ctypes.macholib.dyld
+ctypes.macholib.dylib
+ctypes.macholib.framework
+ctypes.util
+ctypes.wintypes
```

Comment:
I'm not sure the original design was to add submodules like that. I'll defer that to @jackmaney.

Reply:
I can cut out the submodules if that is needed

Reply:
I prefer that. If there is value in adding submodules we can discuss with @jackmaney and do that in another PR.

BTW, right now we use sphinx to create this list. It would be nice to add the import inspection you did here so we can re-do it when a new version is in. (Or issue a release with the same methodology for the older pythons.)

Reply:
I was afraid you were going to ask for that :wink: I'll see what I can do, since I did it in a terminal session.

Reply:
We don't need to do that now ;-p